demos/http/http_demo_s3_download/http_demo_s3_download.c

```diff
@@ -150,81 +150,108 @@
 #define DELAY_BETWEEN_DEMO_RETRY_ITERATIONS_S    ( 5 )
 
 /**
- * @brief Length in bytes of hash digest.
+ * @brief Buffer Length for storing the AWS IoT Credentials retrieved from
+ * AWS IoT credential provider which includes the following:
+ * 1. Access Key ID
+ * 2. Secret Access key
+ * 3. Session Token
+ * 4. Expiration Date
  */
-#define SHA256_HASH_DIGEST_LENGTH                ( ( ( uint16_t ) 64 ) )
+#define CREDENTIAL_BUFFER_LENGTH                 2048U
```

Comment:
Is it required to allocate 2K bytes for it? Wouldn't a much smaller buffer size suffice?